sidebars.js

```diff
@@ -72,6 +72,7 @@ module.exports = {
         "components/concepts/backups",
         "components/concepts/resource-deletion",
         "components/concepts/resource-authorizations",
+        "components/concepts/user-restrictions",
```

Comment:
Hi @akeller, for some reason this setup is not working, Am I missing some detail over here?

Reply:
I think you just want this to live in the Self-Managed section, as shown on line 889. I removed line 75 in my latest commit 👍

Reply:
Thank you